src/sidebar/util/test/annotation-metadata-test.js

```diff
@@ -4,88 +4,96 @@ import * as annotationMetadata from '../annotation-metadata';
 const documentMetadata = annotationMetadata.documentMetadata;
 const domainAndTitle = annotationMetadata.domainAndTitle;
 
-describe('annotation-metadata', function () {
-  describe('.documentMetadata', function () {
-    context('when the model has a document property', function () {
-      it('returns the hostname from model.uri as the domain', function () {
-        const model = {
-          document: {},
-          uri: 'http://example.com/',
-        };
-
+describe('sidebar/util/annotation-metadata', () => {
+  const fakeAnnotation = (props = {}) => {
+    return {
+      document: {},
+      uri: 'http://example.com/a/page',
+      ...props,
+    };
+  };
+
+  describe('documentMetadata', () => {
+    context('when the model has a document property', () => {
+      it('returns the hostname from model.uri as the domain', () => {
+        const model = fakeAnnotation();
```

Comment:
While you're here it might be a good idea to change the `model` variable name to `annotation` or `ann`, since we don't tend to use the generic term "domain model" anywhere in the code to refer to annotations and it makes the test descriptions sound odd.